mypy/typeanal.py

```diff
@@ -458,11 +458,16 @@ def pack_paramspec_args(self, an_args: Sequence[Type]) -> list[Type]:
         # These do not support mypy_extensions VarArgs, etc. as they were already analyzed
         # TODO: should these be re-analyzed to get rid of this inconsistency?
         count = len(an_args)
-        if count > 0:
-            first_arg = get_proper_type(an_args[0])
-            if not (count == 1 and isinstance(first_arg, (Parameters, ParamSpecType, AnyType))):
-                return [Parameters(an_args, [ARG_POS] * count, [None] * count)]
-        return list(an_args)
+        if count == 0:
+            return []
+        if count == 1 and isinstance(get_proper_type(an_args[0]), AnyType):
+            # Single Any is interpreted as ..., rather that a single argument with Any type.
+            # I didn't find this in the PEP, but it sounds reasonable.
+            return list(an_args)
+        if any(isinstance(a, (Parameters, ParamSpecType)) for a in an_args):
+            # Nested parameter specifications are not allowed.
+            return list(an_args)
```

Comment:
Another option might be to generate an error and return a list with a single any type.

Reply:
Yeah, currently this will say something like "Too many arguments", I will check if we can make error messages better on this.